<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Matching</class>
 <widget class="QWidget" name="Matching">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbGrabImage">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Grab Image</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>641</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vlHwindows"/>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Model parameter</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>160</width>
       <height>541</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QComboBox" name="cmbRoiShape">
          <item>
           <property name="text">
            <string>Rectangle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rectangle 2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Circle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ellispe</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Polygone</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Free hand</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pbAddRoi">
          <property name="text">
           <string>Add ROI</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pbDeleteRoi">
          <property name="text">
           <string>Delete ROI</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pbAddAll">
          <property name="text">
           <string>Add All</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pbDeleteAll">
          <property name="text">
           <string>Delete All</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pbUndoRoi">
          <property name="text">
           <string>Undo</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pbRedoRoi">
          <property name="text">
           <string>Redo</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pbOkRoi">
          <property name="text">
           <string>Create model</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pbFindModel">
          <property name="text">
           <string>Find model</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lblCameras">
        <property name="text">
         <string>Camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbCamera"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>156</width>
             <height>294</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Parameter 1</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>141</width>
              <height>101</height>
             </rect>
            </property>
            <property name="title">
             <string>Contrast</string>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>128</width>
               <height>74</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Lower threshold</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinLowerThreshold">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Upper threshold</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinUpperThreshold">
                <property name="minimum">
                 <number>2</number>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>20</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Min size</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="spinBoxMinSize">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>30</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>100</y>
              <width>141</width>
              <height>111</height>
             </rect>
            </property>
            <property name="title">
             <string>Angle</string>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>7</x>
               <y>20</y>
               <width>79</width>
               <height>74</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinStartAngle">
                <property name="minimum">
                 <number>-360</number>
                </property>
                <property name="maximum">
                 <number>360</number>
                </property>
                <property name="value">
                 <number>-180</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>End</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinEndAngle">
                <property name="minimum">
                 <number>-360</number>
                </property>
                <property name="maximum">
                 <number>360</number>
                </property>
                <property name="value">
                 <number>180</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Step</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="spinStepAngle">
                <property name="minimum">
                 <number>0</number>
                </property>
                <property name="maximum">
                 <number>11</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>210</y>
              <width>120</width>
              <height>80</height>
             </rect>
            </property>
            <property name="title">
             <string>Scale %</string>
            </property>
            <widget class="QLabel" name="label_11">
             <property name="geometry">
              <rect>
               <x>7</x>
               <y>20</y>
               <width>71</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Min</string>
             </property>
            </widget>
            <widget class="QSpinBox" name="spinMinScale">
             <property name="geometry">
              <rect>
               <x>70</x>
               <y>20</y>
               <width>43</width>
               <height>20</height>
              </rect>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>90</number>
             </property>
            </widget>
            <widget class="QSpinBox" name="spinMaxScale">
             <property name="geometry">
              <rect>
               <x>70</x>
               <y>50</y>
               <width>43</width>
               <height>20</height>
              </rect>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>150</number>
             </property>
             <property name="value">
              <number>110</number>
             </property>
            </widget>
            <widget class="QLabel" name="label_12">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>50</y>
               <width>41</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Max</string>
             </property>
            </widget>
           </widget>
          </widget>
          <widget class="QWidget" name="page_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>156</width>
             <height>294</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Parameter 2</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>119</width>
              <height>100</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Min score</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" rowspan="2">
              <widget class="QSpinBox" name="spinMinScore">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>90</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0" rowspan="2">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Num. Matches</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinNumMatchs">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Max overlap</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="spinMaxOverlap">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="singleStep">
                <number>1</number>
               </property>
               <property name="value">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Greediness</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QSpinBox" name="spinGreediness">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>90</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>170</y>
              <width>149</width>
              <height>41</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Subpixel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cobSubpixel">
               <item>
                <property name="text">
                 <string>none</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>interpolation</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>least_squares</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>least_squares_high</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>least_squares_very_high</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>max_deformation 1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>max_deformation 2</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Models</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbModify">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Modify</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCopy">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbDeleteModel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Model name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>12345678912345</string>
      </property>
      <property name="maxLength">
       <number>16</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Save model</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
